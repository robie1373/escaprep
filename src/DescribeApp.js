import React, { Component } from 'react';


class DescribeApp extends Component {
  render () {
    return (
      <div id="appDescription">
        <p>This page is used to import a single csv file from Phishme, rearrange the columns to meet mail merge in Outlook's expectations and allow the user to download the new file.</p>
        <p>To start over, refresh the page.</p>
      </div>
    );
  }
}

export default DescribeApp;
