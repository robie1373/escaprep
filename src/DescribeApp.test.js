import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import DescribeApp from './DescribeApp';

test('snapshot', () => {
  const component = renderer.create(<DescribeApp />);

  let tree = component.toJSON()
  ;
  expect(tree).toMatchSnapshot();
});
