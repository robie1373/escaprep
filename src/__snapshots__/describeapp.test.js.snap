// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  id="appDescription"
>
  <p>
    This page is used to import a single csv file from Phishme or Jim's script, rearrange the columns to meet the expectations of mail merge in Outlook and allow the user to download the new file.
  </p>
  <p>
    To start over, refresh the page.
  </p>
  <p>
    For the next step to work the primary recipient's email address must be in the first column. The rest of the columns can be in any order that makes sense to you.
  </p>
  <p>
    Once you are done, you can go 
    <a
      href="http://www.statelibraryofiowa.org/ld/q-s/silo/e-mail/outlook/email-merge"
    >
      here
    </a>
     for instructions on how to do a mail merge.
  </p>
</div>
`;
